api/src/tooltool_api/api.yml

```diff
@@ -322,14 +322,6 @@ paths:
           in: path
           required: true
           type: string
-        - name: region
```

Comment:
Should we also change [the clients](https://github.com/mozilla-releng/tooltool/blob/7a8467fa2d0b38d4aaad3c566858cb800b524a27/client/tooltool.py#L722) to stop using this parameter? 
Or maybe gracefully ignore the parameter because we don't have full control of the client deployment?

Reply:
Once we switch to cloudfront mode, regions don't make sense. We could accept it and ignore it, I guess, but I couldn't find anything actually using it, so I was thinking we'd remove it and fix clients if they bust.

Reply:
Sounds like a plan!